src/js/rx/containers/RxRefillsApp.jsx

```diff
@@ -1,13 +1,14 @@
 import React from 'react';
 import { connect } from 'react-redux';
 
-import RequiredLoginView from '../../common/components/RequiredLoginView';
 import { closeDisclaimer } from '../actions/disclaimer';
 import { closeRefillModal, closeGlossaryModal } from '../actions/modals';
 import { refillPrescription } from '../actions/prescriptions';
-import Disclaimer from '../components/Disclaimer';
+import Breadcrumbs from '../components/Breadcrumbs';
 import ConfirmRefillModal from '../components/ConfirmRefillModal';
+import Disclaimer from '../components/Disclaimer';
 import GlossaryModal from '../components/GlossaryModal';
+import RequiredLoginView from '../../common/components/RequiredLoginView';
```

Comment:
Generally it's been by backwards depth in relative paths and then alphabetical. Might not be consistent across every module right now, but that's at least what I've been trying to do.